cli/install/install.go

```diff
@@ -501,21 +564,18 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 	}
 	// Build tt.
 	log.Infof("Building tt...")
-	err = util.ExecuteCommand("mage", installCtx.verbose, logFile, path,
-		"build")
+	err = util.ExecuteCommand("mage", installCtx.verbose, logFile, path, "build")
 	if err != nil {
 		printLog(logFile.Name())
 		return err
 	}
-
```

Comment:
These changes are not related to the task (current commit). Please, create a separate commit or undo them.

Reply:
fixed

Reply:
Looks like, not fixed yet.